### PR TITLE
 Add Python 3.6, drop Python 3.3

### LINKED ISSUE #3: Add support for Python 3.6, drop Python 3.3.

### PR DESCRIPTION
**Pull request by jamadden**
Fixes #3.

Built on #4 because otherwise the tests don't pass anywhere.

Project gardening:

- Add coverage environment and coveralls.
  - Reach 100% coverage.
- Rename .txt -> .rst
- Add badges to README.rst
- Universal wheels.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+.. image:: https://travis-ci.org/zopefoundation/z3c.ptcompat.svg?branch=master
+        :target: https://travis-ci.org/zopefoundation/z3c.ptcompat
+
+.. image:: https://coveralls.io/repos/github/zopefoundation/z3c.ptcompat/badge.svg?branch=master
+        :target: https://coveralls.io/github/zopefoundation/z3c.ptcompat?branch=master
+
+This package provides a page template engine implementation based on
+Chameleon. It plugs into the `zope.pagetemplate
+<http://pypi.python.org/pypi/zope.pagetemplate>`_ package and has an
```
Comment (mgedmin) [inline README.rst]: http -> https

**Review by mgedmin — APPROVED** [review]
LGTM

File: README.rst (hunk 1 above)
Comment (jamadden) [reply README.rst]: OK.

**Comment by jamadden** [comment]
Thank you!

Could we please get a PyPI release when convenient (or access)? @icemac is an owner.

**Comment by icemac** [comment]
@jamadden I added you as owner of this package on PyPI.

**Comment by jamadden** [comment]
Thank you!

2.1.0 has been released to PyPI.

**Pull request merged**